src/nti/externalization/externalization.py

```diff
@@ -417,8 +416,8 @@ def choose_field(result, self, ext_name,
 
         if value is not None:
             # If the creator is the system user, catch it here
-            if ext_name is StandardExternalFields_CREATOR and is_system_user(value):
-                value = to_unicode(SYSTEM_USER_NAME)
+            if ext_name == StandardExternalFields_CREATOR and is_system_user(value):
```

Comment:
'is' with strings is tricky on PyPy. They don't guarantee identity in the same way CPython does.